cmd/convert.go

```diff
@@ -46,6 +46,7 @@ var convertCmd = &cobra.Command{
 			ToStdout:               ConvertStdout,
 			CreateChart:            ConvertChart,
 			GenerateYaml:           ConvertYaml,
+			GenerateJson:           ConvertJson,
```

Comment:
I think we can get rid of `ConvertYaml` in code everywhere, since we are not checking on it anymore.